src/ReaderScreen/utils/gutkaScript.js

```diff
@@ -15,6 +15,26 @@ let curPosition = 0;
 let isScrolling;
 let isManuallyScrolling = false;
 
+(function() {
+  // Keep WebView active
+  setInterval(function() {
+    window.ReactNativeWebView.postMessage('heartbeat');
+  }, 30000);
+  
+  // Handle visibility changes
+  document.addEventListener('visibilitychange', function() {
+    if (document.visibilityState === 'visible') {
+      window.ReactNativeWebView.postMessage('visible');
+    }
+  });
+  
+  // Prevent WebView from going to sleep
+  function keepAlive() {
+    window.setTimeout(keepAlive, 30000);
+  }
```

Comment:
These timers run without cleanup, causing memory leaks when the WebView unmounts or reloads. Each WebView instance creates new timers that aren't cleared.

How about if we store timer references and add cleanup function that can be called from React Native when component unmounts?

Reply:
done.